x-pack/metricbeat/module/gcp/metrics/metricset.go

```diff
@@ -143,6 +143,10 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	} else {
 		return m, fmt.Errorf("no credentials_file_path or credentials_json specified")
 	}
+	if m.config.Endpoint != "" {
+		m.Logger().Warnf("You are using a custom endpoint '%s' for the GCP API calls.", m.config.Endpoint)
+		m.config.opt = append(m.config.opt, option.WithEndpoint(m.config.Endpoint))
+	}
```

Comment:
Add a new line to improve readability.

```suggestion

	if m.config.Endpoint != "" {
		m.Logger().Warnf("You are using a custom endpoint '%s' for the GCP API calls.", m.config.Endpoint)
		m.config.opt = append(m.config.opt, option.WithEndpoint(m.config.Endpoint))
	}
```